lib/index.js

```diff
@@ -30,8 +30,16 @@ var Iterable = module.exports = integration('Iterable')
  */
 
 Iterable.prototype.track = function(payload, fn){
+  var endpoint;
+  if (payload["isUpdateCart"]) {
```

Comment:
Yup! Each of these events should be extracted out to discrete methods `Iterable.prototype.addedProduct`, `Iterable.prototype.completedOrder` etc. Each can have its own corresponding exported mapper method as well — that will all be wired up automatically